Makefile

```diff
@@ -60,6 +60,10 @@ unit-verbose:
 ## Linting
 ## --------------------------------------
 
+.PHONY: precommit
```

Comment:
I added it to this PR, ´cause I wanted to check the pre-commit before actually commit, and... well... seems useful.